### PR TITLE
Replace missing images

### PR DESCRIPTION
**Pull request by perrauda**

### CODE REVIEW

**Review by joschne — APPROVED** [review]
perfect!

File: packages/design-system-web/src/components/geov-carousel/geov-carousel.stories.tsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { docsTemlpate } from '../../../.storybook/templates/docsTemplate';
 import { stencilWrapper } from '../../../.storybook/lib/stencilWrapper';
 import componentApi from './docs-component-api.md?raw';
 import overview from './docs-overview.md?raw';
+import { image1, image2, image3, image4, image5, image6, image7, image8, image9, image10, image11, image12 } from './images';
```
Comment (flicksolutions) [inline packages/design-system-web/src/components/geov-carousel/geov-carousel.stories.tsx:6]:
```suggestion
import * as images from './images';
```

File: packages/design-system-web/src/components/geov-carousel/images/index.js
Comment (flicksolutions) [inline packages/design-system-web/src/components/geov-carousel/images/index.js:1]: Why did you create this index file in the first if the images are just importet and used in the .stories.tsx file?

Why not import the images directly in the .stories.tsx-file?

**Review by flicksolutions — COMMENTED** [review]
I think one of two simplifications should be done: Either import the images directly in the stories-file or if not, at least simplify the import in the stories-file.

I'm not blocking the merge tho... since it works.

File: packages/design-system-web/src/components/geov-carousel/images/index.js
Comment (perrauda) [reply packages/design-system-web/src/components/geov-carousel/images/index.js:1]: Without this file, I cannot import images in geov-carousel.stories.tsx... I don't find any other way to export images.

File: packages/design-system-web/src/components/geov-carousel/geov-carousel.stories.tsx [hunk 2]
```diff
@@ -17,19 +19,6 @@ export default {
...
-      'https://www.geovistory.org/carousel-5_quetzalcoatl.jpg',
-      'https://www.geovistory.org/carousel-6_bronzes-benin.jpg',
-      'https://www.geovistory.org/carousel-7_carte-amerique.jpg',
-      'https://www.geovistory.org/carousel-8_assemble-vizille.jpg',
-      'https://www.geovistory.org/carousel-9_acte-deces.jpg',
-      'https://www.geovistory.org/carousel-10_guerre-russo-japonaise.jpg',
-      'https://www.geovistory.org/carousel-11_congret-musique.jpg',
-      'https://www.geovistory.org/carousel-12_toronto.jpg',
-    ]}
+    images={[image1, image2, image3, image4, image5, image6, image7, image8, image9, image10, image11, image12]}
```
[the remark this answers is not in the document]
Comment (perrauda) [reply packages/design-system-web/src/components/geov-carousel/geov-carousel.stories.tsx:22]: Done, thank you

**Review by flicksolutions — APPROVED** [review]

**Pull request merged**